setup.sh

```diff
@@ -175,6 +174,7 @@ run() {
     ## Add bin directories of the suites to path
     for suite in $(find suites -maxdepth 1 -type d -exec basename {} \;); do
         export PATH="${ROFI_BUILD_DIR}/$suite/bin:$PATH"
+        export PYTHONPATH="${ROFI_BUILD_DIR}/$suite/lib:$PYTHONPATH"
```

Comment:
Are you sure this won't break when you want to use python with two compiled suites?
If it breaks, I can't think of a solution, just mention it somewhere

Reply:
What do you mean? What exactly should break?

Reply:
I guess that you look for the `librofiHal.so` in the path. So potentially if we want to use the Python bindings for both desktop and firmware, how would you specify which version to use?

I probably just don't have the full picture of how you want to link the two different libraries. So this might be completely fine.

Reply:
Then there is a collision and the order is unspecified. However, that is true for every other binary in the project. I think it is reasonable to assume that no two suites yield the same-named binary.